### DISCUSSION TITLE
Nicer started server on output

### DISCUSSION THREAD

**Discussion by prisis** [vercel/next.js#18449] [Ideas]
# Feature request

## Is your feature request related to a problem? Please describe.

Im kind of missing the information how to access next.js from my local network

## Describe the solution you'd like

Extend the next-start and next-dev with more information about the server  
  
Example

```
You can view the application in browser.
Local: 0.0.0.0:3000
On Your Network: 111.111.111.111:3000
```

## Describe alternatives you've considered

N/A

## Additional context

Would you accept a MR like this?

**Comment by M1ck0** [comment]
I think that using `ifconfig` for Linux and Mac or `ipconfig` for Windows is a really easy way for finding your local IP address and opening a project on your local network.

Someone from the team might find this useful and implement it.

**Comment by prisis** [comment]
Than i can prepare a merge request for this feature :)
